src/Employee/EmployeeBook.java

```diff
@@ -1,7 +1,11 @@
 package Employee;
 
+import java.text.NumberFormat;
+import java.util.Arrays;
+
 public class EmployeeBook {
     private static Employee[] employees = new Employee[10];
```

Comment:
```suggestion
    private Employee[] employees = new Employee[10];
```
массив не должен быть статическим, потому что должен быть у каждого экземпляра книги свой, в задании про это написано